renderer/src/components/ActivityChartCard.jsx

```
@@ -106,119 +108,85 @@
+        <div>
+          <h3 className="text-lg font-semibold text-gray-800">{title}</h3>
+          {description && (
+            <p className="text-sm text-gray-600 mt-1">{description}</p>
+          )}
+        </div>
+        <div className="flex space-x-2">
+          <button
+            className="p-2 text-gray-600 hover:bg-gray-100 rounded-md transition-colors"
+            onClick={() => generateChart(true)}
+            disabled={loading}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)